### ISSUE TITLE
Add periodic self-tests with reporting of filter failure

### ISSUE THREAD

**Issue by dantheta** [openrightsgroup/blocked-org-uk#257]
Currently, the probes will refuse to start up if the expected sites are not blocked.  However, this is not fed back to the server.  Add a period check (30mins or 1hr) to run the self-test and post the result back to the server, so that the filter-enabled flag on the probe status page can be updated.

**Comment by JimKillock** [comment]
The reverse would also need to be true, for instance on TT or Plusnet: it's possible filters will enable themselves and start giving unexpected results. 

We should also record the periods of unexpected filter up / down time so that we can present filter error stats.

**Comment by edjw** [comment]
What's the status of this? Seems like something we'd have sorted out by now?

**Comment by dantheta** [comment]
Yep - SimonStJG contributed server and client code to do these.  There was one part which was blocked on one of my tickets.  We don't have central visibility of the filter availability, because it's still handled in the probe (with periodic self-tests there).

**Comment by edjw** [comment]
What should we do now on this in that case?
